### ISSUE TITLE
Inconsistent Redirection

### ISSUE THREAD

**Issue by redeye25** [johngodley/redirection#2177]
**Intended Behaviour**
Test site mimics my live site. 
WooCommerce website + LoginPress for custom login page . LoginPress only changes WP login not Woo Login page through My Account. 

Therefore I have tried to use Redirection to redirect /my-account -> /login
Logged in users: Not redirected (as they need to access /my-account to view orders)
Logged out users: Redirected to /login (then LoginPress will redirect them back to /my-account after successful login).

**Current Behaviour**
The /my-account redirection is not consistent. Sometimes it will redirect properly to /login, and other times it won't. 

Most often this happens when you log in > log out > try to access /my-account then you get the standard Woo Login rather than being redirected to the login page.

Environment:
Google Chrome and Firefox tested
WP v5.3.2
Redirection v4.6.2

Redirection settings
![image](https://user-images.githubusercontent.com/60996316/74400372-91612800-4e82-11ea-80f2-6fb10595e0e8.png)

LoginPress settings
![image](https://user-images.githubusercontent.com/60996316/74400405-accc3300-4e82-11ea-9979-7e2625fd3fe7.png)

Video of what is happening:
https://d.pr/v/Y7m6L2

**Comment by johngodley** [comment]
I'm not really sure what the video is showing, and I have no knowledge of Loginpress.

Redirection doesn't randomly change behaviour. Usually if something isn't working it is a cache. Would that apply?

**Comment by redeye25** [comment]
Hi John,

Apologies - I was showing in the video that if you type in /my-account after logging out, it will send you back to the /my-account page rather than being redirected to /login (the blue login page) as it's set to do.

However, from your suggestion I did go hunting in the Redirection cache settings (the site was a test site so no caching plugin) and I set the Redirection cache to "Forever" and now it is working as expected. I'm not sure why setting it to 1 day/1 week cache doesn't work though, especially if I'm not leaving the site, but nevermind. Seems to be working now with Forever option so I'll just leave it at that.

Thanks for the response.
